theme/scripts/nav.ts

```diff
@@ -0,0 +1,14 @@
+const navOpenMenuButton = document.getElementById('nav-open-menu-button');
```

Comment:
One thing that we did on Nocta site is we tried to not use a specific element name in the script. We tried to describe what it does instead. So in this case we will use the name `activatable` things that would be activated by click. Then we can use this script in other cases as well. I am not too sure the plan with this app but I would recommend that method!

Reply:
Thanks Sohee! I don't totally understand how this works so I'll ask you on twist or implement this in a pt 2!